src/actions/fetch/copier.ts

```diff
@@ -217,7 +151,7 @@ export function createFetchCopierAction(options: {
         outputPath: templateContentsDir,
```

Comment:
Do we even need to run `fetchContents` anymore?

Reply:
Yes. That reaches for the catalog-info file for the UI